test/test.scss

```diff
@@ -0,0 +1,12 @@
+@import '../table.scss';
```

Comment:
I think `table.css.scss` already includes `table.scss`, yes?  Also, is it possible to just use the selectors produced by by `table.css.scss`?  If not, then I wonder why both including it?